packages/wallet/backend/src/user/service.ts

```diff
@@ -72,7 +72,7 @@ export class UserService implements IUserService {
 
     const resetToken = getRandomToken()
     const passwordResetToken = hashToken(resetToken)
-    const passwordResetExpiresAt = new Date(Date.now() + 10 * 60 * 1000)
+    const passwordResetExpiresAt = new Date(Date.now() + 12 * 3600 * 1000)
```

Comment:
Not to keen about this. I feel like having the password reset expiry at 12 hours might be a bit too much, with the fact that we are going to have a real production environment as well.

I feel like 30 minutes, 1 hour or 2 hours - one of these options, would probably fit best. 


We should also mention in the email for how long the reset link is going to be available - somewhere around here I would say:

https://github.com/interledger/testnet/blob/60eb19321277f14c0fe44819ef443d94e62fef05/packages/wallet/backend/src/email/templates/forgotPassword.ts#L30-L41

CC @Tymmmy @beniaminmunteanu @dragosp1011

Reply:
Feedback directly from Brianna through Tseli, we should have this around half a day, let's do 8 working hours then. But for sure not less. @raducristianpopa

Reply:
updated to 8h and set notification message in the email